integrations/access/opsgenie/app.go

```diff
@@ -429,7 +439,7 @@ func (a *App) tryApproveRequest(ctx context.Context, req types.AccessRequest) er
 		if _, err := a.teleport.SubmitAccessReview(ctx, types.AccessReviewSubmission{
 			RequestID: req.GetName(),
 			Review: types.AccessReview{
-				Author:        tp.SystemAccessApproverUserName,
+				Author:        a.conf.TeleportUserName,
```

Comment:
Looking at your [other PR](https://github.com/gravitational/teleport.e/pull/3774/files#diff-7c919d60e8a9363ba673474c3a99a1fe393e36aa3374b43d76d9ae0c9c7c27ebR38), it looks like it's still set to `teleport.SystemAccessApproverUserName` there, right? What's the reason for replacing the constant with the config field here then, I don't think it's supposed to be configurable, or am I missing something?

Reply:
Since I needed to change opsgenie.go in the other PR to correctly call instantiate the opsgenie plugin i swapped this to be configured there to match Pagerduty and jira.  I can swap it back to the constant instead if you think that makes more sense. This change was for consistency more than anything else.

Reply:
Got it, that's fine. Let's keep it.